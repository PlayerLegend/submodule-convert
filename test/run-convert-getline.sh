#!/bin/sh

$DEBUG_PROGRAM test/convert-getline <<EOF
this is a line
this is another line
a
b
c
d
1
2
3
4
more lines
all done
EOF
