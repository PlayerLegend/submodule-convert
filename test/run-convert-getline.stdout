line: this is a line
line: this is another line
line: a
line: b
line: c
line: d
line: 1
line: 2
line: 3
line: 4
line: more lines
line: all done
